src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -162,6 +162,10 @@
     
     private static final String ROBOT_ID_LOC = "/home/lvuser/robot-type.conf";
 
+    
+    // Battery Variable which equals the voltage of the battery
+    public static final double BatteryCapacity = 12.0;
```

Comment:
To use RobotMap properly, this should go with the rest of the constants before `ROBOT_ID_LOC` and look like:
```Java
    // Battery Variable which equals the voltage of the battery
    private static final double[] BATTERY_CAPACITY_VALS = {12.0, 12.0, 12.0};
    public static final double BATTERY_CAPACITY = getConstant("BATTERY_CAPACITY");
```